Logging+/logging+.js

```diff
@@ -0,0 +1,431 @@
+const { MessageEmbed, MessageActionRow, MessageButton } = require('discord.js')
+
+ 
+originalConfig = require('../config.json')
+let storeDomain = originalConfig.siteInformation.domain
+
+
+module.exports = async function (app, connection, bot, faxstore, client) {
+  let logchan = bot.channels.cache.get(originalConfig.discordConfig.loggingChannelId);
+faxstore.on('login', function (userObject, DbUserResults) {
+
+  const date = Date.now();
+ 
+  const embed = new MessageEmbed()
+    .setTitle(`Logged In`)
+    .setDescription(`**UserId:** \`${DbUserResults.userId}\``)
+    .setTimestamp()
+
+
+    const row = new MessageActionRow()
+    .addComponents(
+      new MessageButton()
+        .setLabel('User Account')
+        .setStyle('LINK')
+        .setURL(`${storeDomain}/user/${DbUserResults.userId}`),
+
+        
+    )
+  logchan.send({ embeds: [embed], components: [row] })
+
+  faxstore.emit('CreateAuditLog', DbUserResults.userId, 'Logged in', `logged in at <t:${date}>`);
+});
+
+faxstore.on('logout', function (userObject) {
+
+    const date = Date.now();
+
+    const embed = new MessageEmbed()
+      .setTitle(`Logged Out`)
+      .setDescription(`**UserId:** \`${userObject.id}\``)
+      .setTimestamp()
+
+
+    const row = new MessageActionRow()
+      .addComponents(
+        new MessageButton()
+          .setLabel('User Account')
+          .setStyle('LINK')
+          .setURL(`${storeDomain}/user/${userObject.id}`),
+      )
+
+    logchan.send({ embeds: [embed], components: [row] })
+
+    faxstore.emit('CreateAuditLog', userObject.id, 'Logged Out', `logged out at <t:${date}>`);
+   
+});
+
+faxstore.on('createUserAccount', function (userObject, serviceType) {
+
+  const date = Date.now();
+
+  const embed = new MessageEmbed()
+    .setTitle(`New User Account`)
+    .setDescription(`**UserId:** \`${userObject.id}\` \n **Login Type:** \`${serviceType}\``)
+    .setTimestamp()
+
+
+  const row = new MessageActionRow()
+    .addComponents(
+      new MessageButton()
+        .setLabel('User Account')
+        .setStyle('LINK')
+        .setURL(`${storeDomain}/user/${userObject.id}`),
+    )
+
+  logchan.send({ embeds: [embed], components: [row] })
+
+  faxstore.emit('CreateAuditLog', userObject.id, 'New User Account', `Account created at <t:${date}> | Login Type: ${serviceType}`);
+});
+
+faxstore.on('onStart', function (licenseKey, siteDomain) {
+
+  const date = Date.now();
+
+  const embed = new MessageEmbed()
+    .setTitle(`Faxstore Started`)
+    .setDescription(`**License Key:** ||\`${licenseKey}\`||`)
```

Comment:
Remove this.

Reply:
Just line 87.

Reply:
Yep, thank you for pointing that out i just removed it